### DISCUSSION TITLE
请问使用工作流发布成github page，经常在Build阶段出错，请问如何解决的

### DISCUSSION THREAD

**Discussion by Halecoder** [oldwinter/knowledge-garden#57] [提问] [answered]
我也看了debug日志，但是日志无法让我定位到错误
![image](https://user-images.githubusercontent.com/75030331/224461582-73c17409-8490-45f4-bff6-b109dbb45895.png)

**Comment by Halecoder** [comment, marked as answer]
好了，排除法找到了。看来需要经常git一下，要不然范围太大了

**Reply by oldwinter** [reply]
是的哈，可能有些语法不支持会导致报错，我也遇到几次。
我看了一下你的花园，局部链接图那边呈现非中文字符，是因为quartz要求frontdata里面要有title字段，并且要用双引号括起来。你如果觉得需要，可以加上哈。

**Reply by Halecoder** [reply]
> 范围

蟹蟹
